hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/DoubleArrayCost.java

```diff
@@ -106,4 +88,32 @@ private static double getSum(double[] stats) {
     }
     return total;
   }
+
+  /**
+   * Return the min skew of distribution
+   */
+  public static double getMinSkew(double total, double numServers) {
```

Comment:
Why are the input arguments `double`? Can there be a fractional amount of either of these quantities?

Reply:
It is to convert the input from integer to double for computation in the function.